ebextensions.pro/nginx.config

```diff
@@ -0,0 +1,88 @@
+                # including file...
+                gzip_types text/plain text/css application/json application/x-javascript text/xml application/xml application/xml+rss text/javascript;
+                if ($time_iso8601 ~ "^(\d{4})-(\d{2})-(\d{2})T(\d{2})") {
+                    set $year $1;
+                    set $month $2;
+                    set $day $3;
+                    set $hour $4;
+                }
+                access_log /var/log/nginx/healthd/application.log.$year-$month-$day-$hour healthd;
+                access_log    /var/log/nginx/access.log;
+                location / {
+
+                    proxy_pass            http://docker;
+                    proxy_http_version    1.1;
+                    proxy_set_header      Connection $connection_upgrade;
+                    proxy_set_header      Upgrade    $http_upgrade;
+                    proxy_set_header      Host       $host;
+                    proxy_set_header      X-Real-IP  $remote_addr;
+                    proxy_set_header      X-Forwarded-For $proxy_add_x_forwarded_for;
+                    proxy_hide_header     ETag;
+                    proxy_hide_header     X-Powered-By;
+
+                    #
+                    # REDIRECTS
+                    #
+
+                    # ... Pro site:
+                    #
+
+                    # Pro Site > Ebooks
+                    rewrite ^/ebooks https://ebooks.dp.la permanent;
+                    rewrite ^/ebooks/ebook-bibliolabs https://ebooks.dp.la/bibliolabs permanent;
+                    rewrite ^/ebooks/curation-corps https://ebooks.dp.la/curation-corps permanent;
+                    rewrite ^/ebooks/open-ebooks-2 https://ebooks.dp.la/open-ebooks permanent;
+                    rewrite ^/ebooks/dpla-exchange https://ebooks.dp.la/dpla-exchange permanent;
+                    rewrite ^/ebooks/dpla-publications https://ebooks.dp.la/dpla-publications permanent;
+                    rewrite ^/ebooks/ebook-simplye https://ebooks.dp.la/simplye permanent;
+                    rewrite ^/ebooks/ebook-platform https://ebooks.dp.la/platform permanent;
+                    rewrite ^/ebooks/ebook-hosting https://ebooks.dp.la/hosting permanent;
+                    rewrite ^/ebooks/faqs https://ebooks.dp.la/faqs permanent;
+                    rewrite ^/ebooks/ebook-investing-in-libraries https://ebooks.dp.la/investing-in-libraries permanent;
+                    rewrite ^/ebooks/ebook-community https://ebooks.dp.la/community permanent;
+
+                    location /info/help/accounts {
+                        return 410;
+                    }
+                    location /info/help/glossary {
+                        return 410;
+                    }
+                    location /info/author {
+                        return 410;
+                    }
+                    location /users/sign {  # sign_in and sign_up
+                        return 410;
+                    }
```

Comment:
I don't think we need these. They're left over from the old Rails version of dp.la, which never existed at pro.dp.la